pkg/api/v1/clients/kube/crd/util.go

```diff
@@ -0,0 +1,17 @@
+package crd
+
+import (
+	"github.com/solo-io/solo-kit/pkg/utils/protoutils"
+)
+
+func Copy(src, dst SoloKitCrd) error {
+	srcBytes, err := protoutils.MarshalBytes(src)
+	if err != nil {
+		return err
+	}
+	err = protoutils.UnmarshalBytes(srcBytes, dst)
+	if err != nil {
+		return err
+	}
+	return nil
```

Comment:
why are we doing this copy? there's a resources.Clone function

Reply:
The copy needs to happen in place onto `dst` -- clone returns a new object